README.md

```diff
@@ -7,7 +7,8 @@ Neovim.
   https://github.com/neovim/neovim/commit/3853276d9cacc99a2698117e904475dbf7033383), users will need
   to update to a version of Neovim **after** the aforementioned PR was merged. There are currently
   shims around the changed functions which should maintain support for earlier versions of Neovim,
-  but these are intended to be temporary and will be removed by **2020-10-04**.
+  but these are intended to be temporary and will be removed by **2020-10-04**. Therefore
+  Packer will not work with Neovim v0.4.4, which was relased before the `extmark` change.
```

Comment:
Slight typo here: "relased" should be "released"

Reply:
Fixed